### PR TITLE
Fix JSON Writer docs for data writter as Base64-encoded text

### PR DESCRIPTION
**Pull request by safern**
Ports: https://github.com/dotnet/corefx/pull/40200 and adds consistency.

cc: @mairaw @rpetrusha @ahsonkhan

### CODE REVIEW

File: xml/System.Text.Json/Utf8JsonWriter.xml [hunk 1]
```diff
@@ -444,8 +444,8 @@ The <xref:System.Text.Json.Utf8JsonWriter> will continue to use the original wri
       </Parameters>
       <Docs>
         <param name="utf8PropertyName">The UTF-8 encoded name of the property to write.</param>
-        <param name="bytes">The binary data to be written as a base 64 encoded JSON string as part of the name/value pair.</param>
-        <summary>Writes the property name and raw bytes value (as a base 64 encoded JSON string) as part of a name/value pair of a JSON object.</summary>
+        <param name="bytes">The binary data to write as a Base64 encoded text.</param>
```
Comment (rpetrusha) [inline xml/System.Text.Json/Utf8JsonWriter.xml]:
```suggestion
        <param name="bytes">The binary data to write as Base64 encoded text.</param>
```

File: xml/System.Text.Json/Utf8JsonWriter.xml [hunk 2]
```diff
@@ -516,7 +516,7 @@ The property name is escaped before writing.
       </Parameters>
       <Docs>
         <param name="propertyName">The property name of the JSON object to be transcoded and written as UTF-8.</param>
-        <param name="bytes">The binary data to be written as a base 64 encoded JSON string as part of the name/value pair.</param>
+        <param name="bytes">The binary data to write as a Base64 encoded text.</param>
```
Comment (rpetrusha) [inline xml/System.Text.Json/Utf8JsonWriter.xml]:
```suggestion
        <param name="bytes">The binary data to write as Base64 encoded text.</param>
```

File: xml/System.Text.Json/Utf8JsonWriter.xml [hunk 3]
```diff
@@ -553,8 +553,8 @@ The property name is escaped before writing.
       </Parameters>
       <Docs>
         <param name="propertyName">The JSON-encoded name of the property to write.</param>
-        <param name="bytes">The binary data to be written as a base 64 encoded JSON string as part of the name/value pair.</param>
-        <summary>Writes the pre-encoded property name and raw bytes value (as a base 64 encoded JSON string) as part of a name/value pair of a JSON object.</summary>
+        <param name="bytes">The binary data to write as a Base64 encoded text.</param>
```
Comment (rpetrusha) [inline xml/System.Text.Json/Utf8JsonWriter.xml]:
```suggestion
        <param name="bytes">The binary data to write as Base64 encoded text.</param>
```

File: xml/System.Text.Json/Utf8JsonWriter.xml [hunk 4]
```diff
@@ -586,8 +586,8 @@ The property name should already be escaped when the instance of <xref:System.Te
         <Parameter Name="bytes" Type="System.ReadOnlySpan&lt;System.Byte&gt;" />
       </Parameters>
       <Docs>
-        <param name="bytes">The binary data to be written as a base 64 encoded JSON string element of a JSON array.</param>
-        <summary>Writes the raw bytes value as base 64 encoded JSON string as an element of a JSON array.</summary>
+        <param name="bytes">The binary data to be written as a Base64 encoded JSON string element of a JSON array.</param>
+        <summary>Writes the raw bytes value as Base64 encoded JSON string as an element of a JSON array.</summary>
```
Comment (rpetrusha) [inline xml/System.Text.Json/Utf8JsonWriter.xml]:
```suggestion
        <summary>Writes the raw bytes value as a Base64 encoded JSON string as an element of a JSON array.</summary>
```

**Comment by mairaw** [comment]
I feel guilty of giving the wrong guidance in the first place, so I've applied the feedback.

**Comment by safern** [comment]
Thanks @rpetrusha for the feedback and @mairaw for applying it.

File: xml/System.Text.Json/Utf8JsonWriter.xml [hunk 5]
```diff
@@ -586,8 +586,8 @@ The property name should already be escaped when the instance of <xref:System.Te
         <Parameter Name="bytes" Type="System.ReadOnlySpan&lt;System.Byte&gt;" />
       </Parameters>
       <Docs>
-        <param name="bytes">The binary data to be written as a base 64 encoded JSON string element of a JSON array.</param>
-        <summary>Writes the raw bytes value as base 64 encoded JSON string as an element of a JSON array.</summary>
+        <param name="bytes">The binary data to be written as a Base64 encoded JSON string element of a JSON array.</param>
```
Comment (ahsonkhan) [inline xml/System.Text.Json/Utf8JsonWriter.xml]: This should be consistent with the rest:
```suggestion
        <param name="bytes">The binary data to write as Base64 encoded text.</param>
```

File: xml/System.Text.Json/Utf8JsonWriter.xml [hunk 6]
```diff
@@ -586,8 +586,8 @@ The property name should already be escaped when the instance of <xref:System.Te
...
+        <summary>Writes the raw bytes value as a Base64 encoded JSON string as an element of a JSON array.</summary>
```
Comment (ahsonkhan) [inline xml/System.Text.Json/Utf8JsonWriter.xml]: Similarly, this should be consistent with the other docs:
```suggestion
        <summary>Writes the raw bytes value (as a Base64 encoded JSON string) as an element of a JSON array.</summary>
```

**Pull request merged**